vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -66,19 +66,18 @@ ent bundle run conference-details
 ```
 When the run is complete, you should see the details MFE with the ID 1 loaded.
 
-<!-- >>If you want to modify a different row in the database, edit `microfrontends/conference-details/public/index.html` file. Change the `id` attribute in this line:
-
-    <my-entity-details service-url="%REACT_APP_SERVICE_URL%" id="1" />
--->
+>If you want to modify a different row in the database, edit `microfrontends/conference-details/public/index.html` file. Change the `id` attribute in this line:
+``` html
+    <my-entity-details id="1" />
+```
 
 ### Keycloak Settings and Issues 
-<!-- 1. Change Keycloak Dev Settings
-
-If you want to use another Keycloak installation >> injected by the API Claims and config JSON structure.
+1. Change Development Settings
 
- -->
+* If you want to use another Keycloak installation, modify the parameter in the Keycloak YAML file located in the `svc/` folder.
+* To change the `service-url` of the widget, use the [ent bundle CLI](../../../docs/getting-started/ent-api.md) to add the new URL in the API claim. 
```

Comment:
I think we can remove this bullet altogether. That part will be clearer in the blueprint tutorial, I think.